atomix/cluster/src/test/java/io/atomix/cluster/protocol/SwimProtocolTest.java

```diff
@@ -164,7 +211,7 @@ public void testSwimProtocol() throws Exception {
 
     // Verify that neither node is ever removed from the cluster since node 3 can still ping nodes 1
     // and 2.
-    Thread.sleep(5000);
+    Thread.sleep(500);
```

Comment:
Do you think we could get rid of all these `Thread.sleep` :smiley: ? We're clearly waiting for something to happen here - can we use awaitility again for it, or is this something that's not observable?

Reply:
I would love to. But this is not possible since there is no really a thing we can verify. I was really close to just drop the sleeps. And also the last part which says "is never removed" :sweat_smile:

Reply:
If you agree then I would just drop this part.

Reply:
```java

    // Verify that neither node is ever removed from the cluster since node 3 can still ping nodes 1
    // and 2.
    Awaitility.await().atLeast(Duration.ofSeconds(1));
    Thread.sleep(500);
    checkMembers(member1, member1, member2, member3);
    checkMembers(member2, member1, member2, member3);
    checkMembers(member3, member1, member2, member3);
```

Reply:
@npepinpe ping

Reply:
Ah, so it just waits to make sure that nodes are not removed if nothing happens?

Reply:
Yes